### ISSUE TITLE
log-courier not able to install on CentOS version 5

### ISSUE THREAD

**Issue by ajeevpk** [driskell/log-courier#350]
Is log-courier not compatible with RHEL5?

[root@podrbccls3etl01 local_log_courier]# rpm -ivh zeromq3-3.2.5-1.el6.x86_64.rpm
warning: zeromq3-3.2.5-1.el6.x86_64.rpm: Header V3 RSA/SHA256 signature: NOKEY, key ID 0608b895
error: Failed dependencies:
        libc.so.6(GLIBC_2.7)(64bit) is needed by zeromq3-3.2.5-1.el6.x86_64
        libpgm-5.1.so.0()(64bit) is needed by zeromq3-3.2.5-1.el6.x86_64
        libstdc++.so.6(GLIBCXX_3.4.9)(64bit) is needed by zeromq3-3.2.5-1.el6.x86_64
        rpmlib(FileDigests) <= 4.6.0-1 is needed by zeromq3-3.2.5-1.el6.x86_64
        rpmlib(PayloadIsXz) <= 5.2-1 is needed by zeromq3-3.2.5-1.el6.x86_64

**Comment by ajeevpk** [comment]
Both Filebeat and Log Courier not supported on RHEL5, Got reply from elastic community https://discuss.elastic.co/t/is-filebeat-log-courier-and-logstash-supported-for-centos-rhel-5/66740/4

**Comment by driskell** [comment]
The package you attempted to install was for ZeroMQ3 not Log Courier, and it was from EL6 repository so built using binaries on RedHat Enterprise Linux 6.

Log Courier will only just about work on RHEL5 I think but you will need to compile everything manually from scratch (including Go compiler). I do remember many years ago when I started I did compile logstash-forwarder and log-courier for RHEL5 on some legacy systems in our data centre. Those are long gone though. It should be noted this was with a very earlier version and possibly the newer Go compilers that Log Courier needs won't work at all anymore on RHEL5.

It's also worth noting that there are some very strange clock issues in Go runtime sometimes on 32-bit RHEL5 that I encountered and never resolved, though it was only the clock so not a major problem. Log timestamps would just go haywire 100s years into the future.
